### DISCUSSION TITLE
Release policy of NGINX Ingress Controller

### DISCUSSION THREAD

**Discussion by Uvindu96** [nginx/kubernetes-ingress#3121] [Q&A] [answered]
I would like to know what is the release policy for `NGINX Ingress Controller` with the latest K8s versions, What is the policy to support major. minor and patch versions of K8s. And how frequently releases are going to make ?

**Comment by brianehlert** [comment, marked as answer]
This project depends on the Kubernetes API and follows a pattern of Kubernetes conformance.  And strives to stay current with changes to the K8s API that directly impact the API surface that the project uses.

The latest versions of K8s are validated as part of the regular release process which follows a pattern of quarterly.

There is only a tight binding of of this project (or similar) to a particular K8s version when the Kubernetes project deprecates some API version, like was done with Ingress v1beta1 with release 1.22.
